diagrams/carry4-flatten.svg

```diff
@@ -0,0 +1,317 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:s="https://github.com/nturley/netlistsvg" width="511" height="435.5">
```

Comment:
I don't think we need a new folder for this? Let's just put it at the top level or in the ./docs/ directory?